quiz.py

```diff
@@ -2,5 +2,51 @@
 File: quiz.py
 Author: Name
 Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Description: A Multiple choice game that calculates the score
+"""
+print("Multiple-Question Quiz Game (NBA)")
+
+correct_answers = 0
+
+print("1. Who scored the most career points in the NBA?")
+print("(a) Michael Jordan")
+print("(b) Lebron James")
+print("(c) Kareem Abdul-Jabbar")
+
+answer_1 = input("Answer:")
+
+if answer_1.lower() == "b":
```

Comment:
Clever use of the `.lower()` method to force a string to lowercase before evaluating it with the comparison operator. I see you've used three variables to get user input, one for each question. Have you considered just using the same variable again and again? You essentially re-assign the variable every time you write a new `input()` statement. It's more efficient.

Reply:
I have to strongly suggest that you not use the `.lower()` method here and instead solve the issue of case insensitivity with things we actually learned in class. Use [comparison operators](https://github.com/STE-Technology/ICS3U-Unit-2-Control-Flow#comparison-operators) and [logical operators](https://github.com/STE-Technology/ICS3U-Unit-2-Control-Flow#logical-operators) to achieve the same thing.